### PR TITLE
Kyuubi monitor

### PR DESCRIPTION
**Pull request by lxk000000**
lxk000000 [Closes #437](https://github.com/yaooqinn/kyuubi/pull/437)    Target Issue Test Plan [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=yaooqinn&utm_campaign=badge_info)

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by pan3793** [comment]
Can we manage npm deps by maven? Just like
```
<dependency>
    <groupId>org.webjars</groupId>
    <artifactId>jquery</artifactId>
    <version>3.6.0</version>
</dependency>
```

**Comment by yaooqinn** [comment]
Thanks, @lxk000000, this gonna be a huge step for Kyuubi to improve itself, and many users can benefit from this feature. For a better reviewing and merging process, let's make the basic needs clear to every contributor and review.

The first step what we may move forward here is to separate this giant PR into small ones,

First thing first, we can divide the write-path(how and what we output) and the read-path( Kyuubi-web).

For the write-path, we need a more general implementation, and the most important thing here is to define the schema/messages for output, then we can decide how we output.

BTW, Can this PR be built and run locally by all the reviews, can you provide some instructions in the PR description as a quick start guide?

also cc @ulysses-you @zwangsheng

File: kyuubi-dao/pom.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            <artifactId>kyuubi-common</artifactId>
+            <version>1.2.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.kyuubi</groupId>
+            <artifactId>kyuubi-common</artifactId>
+            <version>1.2.0-SNAPSHOT</version>
+            <scope>compile</scope>
+        </dependency>
```
Comment (zwangsheng) [inline kyuubi-dao/pom.xml:104]: there are three same dependencies
